api/overview/usage_examples.adoc

```diff
@@ -0,0 +1,479 @@
+
+[[usage-examples]]
+= Usage Examples
+
+The below examples illustrate how to use the ManageIQ REST API with the Client for URLs (cURL), Ruby and Python.
+All examples shown use the default admin credentials with insecure methods.
+
+Three basic examples will be shown, a **GET**, a **POST** with a simple payload and a 
+**GET** with filtering
+
+All examples use IP Address **192.0.2.0** to represent the ManageIQ appliance, with the default HTTPS port omitted.
+
+'''''
+
+* <<curl,cURL>>
+  - <<curl-example1,Example 1: A simple **GET** with cURL>>
+  - <<curl-example2,Example 2: Using **POST** to edit a group with cURL>>
+  - <<curl-example3,Example 3: A **GET** using filtering with cURL>>
+* <<ruby,Ruby>>
+  - <<ruby-example1,Example 1: A simple **GET** with Ruby>>
+  - <<ruby-example2,Example 2: Using **POST** to edit a group with Ruby>>
+  - <<ruby-example3,Example 3: A **GET** using filtering with Ruby>>
+* <<python,Python>>
+  - <<python-example1,Example 1: A simple **GET** with Python>>
+  - <<python-example2,Example 2: Using **POST** to edit a group with Python>>
+  - <<python-example3,Example 3: A **GET** using filtering with Python>>
+
+'''''
+
+[[curl]]
+== cURL
+
+The cURL command can be used from the shell. However the shell's interpretation of some characters, for example **&**, can make it awkward to use for complex cases.
+To help alleviate this the payload can be passed to cURL using the **--data** flag, where the payload is stored in a file in the format specified 
+in the content-type, **JSON** in our example.
```

Comment:
feels awkward, maybe just "...  the --data flag, where the JSON payload is provided in a file."

Reply:
Agreed. Will do.